src/main/java/org/codewars/utils/Reader.java

```diff
@@ -22,7 +24,22 @@ public static String readString() {
         return "";
     }
     public static int[] readArrInt() {
-        return new int[0];
+        String input = scanner.nextLine().trim();
```

Comment:
mb show smth like "enter whole numbers, separated with space"? It's not clear from user side now